pkg/controller/vpc_nat_gateway.go

```diff
@@ -749,3 +754,61 @@ func (c *Controller) initCreateAt(key string) (err error) {
 	NAT_GW_CREATED_AT = pod.CreationTimestamp.Format("2006-01-02T15:04:05")
 	return nil
 }
+
+func (c *Controller) updateCrdNatGw(key string) error {
+	gwCr, err := c.config.KubeOvnClient.KubeovnV1().VpcNatGateways().Get(context.Background(), key, metav1.GetOptions{})
+	if err != nil {
+		errMsg := fmt.Errorf("failed to get vpc nat gw '%s', %v", key, err)
+		klog.Error(errMsg)
+		return errMsg
+	}
+	gw := gwCr.DeepCopy()
```

Comment:
the resource is from client not cache, no need to deepcopy here

Reply:
fixed